e2e-polybft/framework/test-bridge.go

```diff
@@ -229,7 +229,8 @@ func (t *TestBridge) Withdraw(token bridgeCommon.TokenType,
 			"--sender-key", senderKey,
 			"--receivers", receivers,
 			"--amounts", amounts,
-			"--json-rpc", jsonRPCAddr)
+			"--json-rpc", jsonRPCAddr,
+			"--child-token", childToken.String())
```

Comment:
There is already child-token flag provided a couple of lines before:
```suggestion
			"--json-rpc", jsonRPCAddr)
```

Reply:
Removed